docs/training-guide.md

```diff
@@ -1,17 +1,177 @@
+---
+layout: default
+title: Model training guide
+nav_order: 2
+---
+
 # Model training guide
 
-First of all, choose a language pair to train.
+A step-by-step guide on how to train a translation model. 
 
-## Configuration
-Clone the repo and follow the instructions that correspond to the workflow manager you will be using 
-([Taskcluster](task-cluster.md), [Snakemake](snakemake.md)).
+The configuration of the training run happens mostly in the training configuration file. 
+Look at the examples of the full production configs for
+[Taskcluster](https://github.com/mozilla/firefox-translations-training/tree/main/configs/tc.prod.yml) and 
+[Snakemake](https://github.com/mozilla/firefox-translations-training/tree/main/configs/config.prod.yml).
 
-The Marian workspace is usually safe to set to about 3/4 of available GPU memory 
-(in a [profile for Snakemake](/pipeline/train/train.sh) and throughout the ci steps in Task cluster).
+## 1. Choose a language
+
+First, choose a language pair to train.
+
+Considerations:
+- The size of the parallel corpus on [OPUS](https://opus.nlpl.eu/)
+- Availability of monolingual data. The pipeline requires monolingual data in both source and target languages. 
+  Currently we support automatic donwloading only for [news crawl](https://data.statmt.org/news-crawl/)
+- Availability of [bicleaner-ai models](https://github.com/bitextor/bicleaner-ai-data/releases)
+
+Set the language pair and a name of the experiment in the config:
```

Comment:
Just by reading this, it isn't clear what the "config" is

Reply:
updated